packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -334,7 +347,7 @@ export const resolveBindingNode = (
 
   if (binding.path.parentPath.isImportDeclaration()) {
     if (!meta.state.filename) {
-      throw new Error('Filename is needed for module traversal.');
+      return;
```

Comment:
Removed this error because it was getting thrown when styles are coming from `React.useState`. I thought its better to return early instead of throwing an error. This is such use case:

```typescript
import React from 'react';
import '@compiled/core';

const [fontSize] = React.useState('10px');

const Component = () => <div css={{ fontSize }}>hello world</div>;
```

Reply:
nice!

Reply:
Yups sure. Will add the comment.

Hardcoded 🤔 hmm..just thinking what should we harcode 🤔. Should we hardcode only most used packages like React?

Reply:
I mean what all we will hardcode? May be we can have whitelisting of packages we allow eg. `@atlaskit/*`, `@material-ui/*`, `antd*`, `@adobe/react-spectrum` etc. Few popular design systems or libraries from where theming/coloring are coming from?

Reply:
for now let's not worry about it - but keep it in the back of our mind